packages/core/strapi/lib/services/metrics/admin-user-hash.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const hash = require('hash.js');
+
+const generateAdminUserHash = (ctx) => {
+  if (!ctx?.state?.user) return '';
```

Comment:
Just a quick suggestion to stay consistent with the rest of the codebase

```suggestion
  if (!ctx?.state?.user) {
    return '';
  }
```